pkg/bpf-mgr/lib-bpfman.go

```diff
@@ -0,0 +1,123 @@
+package bpf_mgr
+
+import (
+	"context"
+	"encoding/binary"
+	"fmt"
+
+	"github.com/openshift/ingress-node-firewall/api/v1alpha1"
+
+	bpfmaniov1alpha1 "github.com/bpfman/bpfman-operator/apis/v1alpha1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/klog"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	debugLookup                = "debug_lookup" // constant defined in kernel hook to enable lPM lookup
```

Comment:
If lPM is Longest Prefix Match can we write it out explicitly.

Reply:
the dbg is just to dbg the kep of LPM lookup

Reply:
My observation was really: I don't know what lPM stands for, so we should spell it out explicitly.

Reply:
done